Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -628,6 +630,61 @@ NonnullRefPtr<Range> Document::create_range()
     return Range::create(*this);
 }
 
+// https://dom.spec.whatwg.org/#dom-document-createevent
+NonnullRefPtr<Event> Document::create_event(const String& interface)
+{
+    auto interface_lowecase = interface.to_lowercase();
+    RefPtr<Event> event;
+    if (interface_lowecase == "beforeunloadevent") {
+        event = Event::create(""); // FIXME: Create BeforeUnloadEvent
+    } else if (interface_lowecase == "compositionevent") {
+        event = Event::create(""); // FIXME: Create CompositionEvent
+    } else if (interface_lowecase == "customevent") {
+        event = Event::create(""); // FIXME: Create CustomEvent
+    } else if (interface_lowecase == "devicemotionevent") {
+        event = Event::create(""); // FIXME: Create DeviceMotionEvent
+    } else if (interface_lowecase == "deviceorientationevent") {
+        event = Event::create(""); // FIXME: Create DeviceOrientationEvent
+    } else if (interface.equals_ignoring_case("dragevent")) {
```

Comment:
Accidentally forgot to use the new `interface_lowercase`:
```suggestion
    } else if (interface_lowercase == "dragevent")) {
```